test/minimal_init.lua

```diff
@@ -0,0 +1,73 @@
+local execute = vim.api.nvim_command
+local fn = vim.fn
+
+local install_path = "/tmp/site/pack/packer/opt/packer.nvim"
+
+if fn.empty(fn.glob(install_path)) > 0 then
+  execute("!git clone https://github.com/wbthomason/packer.nvim " .. install_path)
+  execute "packadd packer.nvim"
+end
+
+execute "packadd packer.nvim"
+
+vim.o.termguicolors = true
+
+-- Only required if you have packer in your `opt` pack
+vim.cmd [[packadd packer.nvim]]
+vim.cmd [[autocmd BufWritePost minimal_init.lua PackerCompile]]
+vim.cmd [[autocmd BufWritePost minimal_init.lua PackerInstall]]
```

Comment:
shouldn't this be `install` first and then `compile` ? I thought it needs to install stuff and then compile it.

Reply:
PackerCompile compiles the package lazy loading lists, I believe this has to be done before installing.

Reply:
The readme says `You must run this whenever you make changes to your plugin configuration` so I assume it needs to be done after we change something in the packer config or install something new. I'm not sure to be honest :laughing:

Reply:
oh, I just realised they're autocmd, the order shouldn't make any difference I guess